citrination_client/views/model_template/client.py

```diff
@@ -7,5 +7,47 @@ class ModelTemplateClient(BaseClient):
     """
 
     def __init__(self, api_key, webserver_host="https://citrination.com", suppress_warnings=False, proxies=None):
-        members = ["validate"]
+        members = ["create", "validate"]
         super(ModelTemplateClient, self).__init__(api_key, webserver_host, members, suppress_warnings, proxies)
+
+    def create(self, search_template, ml_config):
```

Comment:
ModelTemplateClient definitely cannot have a create method that references `ml_config`. ml_config is peculiar to a specific style of creating a template.

Reply:
That is needed by the endpoint to make the model template, we can internally create the ml config based on something else, but how do we get the descriptor info into the endpoint to make the modeltemplate?

Reply:
`ModelTemplateClient` doesn't need to be the only thing that hits an endpoint defined in the `ModelTemplateController`. 

If it makes you feel better, we can refactor the scala code to make the Client<->Controller relationship more clear. I'm focused on getting the API/interface right, then propagating that back to the code for our sanity later.

Reply:
To be clear - my suggestion is that this class _not_ have a create method.